nixos/desktop/default.nix

```diff
@@ -102,7 +102,7 @@
   # https://github.com/NixOS/nixpkgs/issues/101616
   #
   # Using mkforce for https://discourse.nixos.org/t/gpg-smartcard-for-ssh/33689/3
-  services.gnome.gnome-keyring.enable = lib.mkForce false;
+  # services.gnome.gnome-keyring.enable = lib.mkForce false;
```

Comment:
Can I simply resolve this after https://github.com/NixOS/nixpkgs/pull/310978?

Reply:
I should remember why I should avoid gnome-keyring in SSH_AUTH_SOCK 

https://github.com/kachick/dotfiles/issues/714
https://github.com/NixOS/nixpkgs/issues/140824